pkg/binding/format/format_test.go

```diff
@@ -49,7 +49,7 @@ func TestMarshalUnmarshal(t *testing.T) {
 	assert.NoError(e.SetData("foo"))
 	b, err := format.Marshal(format.JSON.MediaType(), e)
 	assert.NoError(err)
-	assert.Equal(`{"data":"foo","datacontenttype":"application/json","id":"id","source":"source","specversion":"0.3","type":"type"}`, string(b))
+	assert.Equal(`{"data":"foo","id":"id","source":"source","specversion":"0.3","type":"type"}`, string(b))
```

Comment:
@nachocano see this PR:

https://github.com/cloudevents/sdk-go/pull/235/files

(sorry, seeing this just now ..)

Reply:
thanks @matzew , just rebased!